addon/components/freestyle-md-text.js

```diff
@@ -1,15 +0,0 @@
-import Ember from 'ember';
-import MDTextComponent from 'ember-remarkable/components/md-text';
-
-const { computed } = Ember;
-
-export default MDTextComponent.extend({
```

Comment:
This component never made it into a released version. Earlier versions of freestyle were using the md-text component provided by (my fork of) ember-remarkable.

I switched over to this approach to get off of my ember-remarkable fork before realizing ember-cli-showdown + highlight.js-via-CDN would come together so nicely.